test/fixtures/AdapterTests.jl

```diff
@@ -0,0 +1,142 @@
+module AdapterTests
```

Comment:
This is not a test fixture so it should go to some other folder maybe test-helpers testers julia-tester etc.

Reply:
This will be addressed in a later update and is now tracked in #191.